fdbclient/ServerKnobs.cpp

```diff
@@ -488,9 +488,10 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	// Use a smaller memtable in simulation to avoid OOMs.
 	int64_t memtableBytes = isSimulated ? 1024 * 1024 : 512 * 1024 * 1024;
 	init( ROCKSDB_MEMTABLE_BYTES,                      memtableBytes );
-	init( ROCKSDB_LEVEL_STYLE_COMPACTION,                       true );
```

Comment:
Is it safe to just remove knobs? Is there a deprecation cycle for knobs?

Reply:
I think it should be ok because we don't maintain it state anywhere. But if any cluster is manually set this knob, then it might throw some errors ' like knob not found'.

This knob is removed because this knob cannot be changed, once DB is created. Initially added to experiment it. And, I don't want anyone to change it accidentally.